server/src/test/java/org/elasticsearch/index/mapper/BinaryFieldMapperTests.java

```diff
@@ -32,28 +32,18 @@
 import org.elasticsearch.common.xcontent.XContentFactory;
 import org.elasticsearch.common.xcontent.XContentType;
 import org.elasticsearch.plugins.Plugin;
+import org.elasticsearch.test.ESSingleNodeTestCase;
 import org.elasticsearch.test.InternalSettingsPlugin;
 
 import java.io.IOException;
 import java.util.Arrays;
 import java.util.Collection;
-import java.util.Set;
 
 import static org.hamcrest.Matchers.containsString;
 import static org.hamcrest.Matchers.equalTo;
 import static org.hamcrest.Matchers.instanceOf;
 
-public class BinaryFieldMapperTests extends FieldMapperTestCase<BinaryFieldMapper.Builder> {
```

Comment:
Without this we don't test `store` and stuff like that, right?

Reply:
Ah! So you have the tests in `ParameterizedFieldTest` for things like this. Do you think it is important to also test that the field merges here too? I guess not, but I'm worried about dropping something accidentally.

Reply:
The idea would be that we don't have 'common' fields any more - all mappers declare all their parameters outright (I am sharing the 'meta' parameter at the moment, but I think I'm going to stop doing that).  So the tests for each field mapper should cover all the parameters that they declare, but general things like 'this parameter can't be updated so it should throw an error on merge' are tested in `ParametrizedFieldTest`.